src/main/kotlin/lotto/LottoManager.kt

```diff
@@ -0,0 +1,16 @@
+package lotto
+
+class LottoManager(
+    val lotto: Lotto
+) {
+
+    fun buyLotto(money: Int): List<List<Int>> {
```

Comment:
```suggestion
    fun buyLotto(money: Int): List<Lotto> {
```
로또 번호의 리스트를 Lotto 클래스로 관리를 하고 계십니다.
그렇다면, 구현하신 Lotto 클래스를 활용해보면 어떨까요?

Reply:
Lotto 로 수정하였습니다~! 해당 과정에서 변경되는 코드 및 테스트 코드 수정하였습니다!